js/tetris.js

```diff
@@ -87,19 +107,19 @@ function freeze() {
 
 // returns rotates the rotated shape 'current' perpendicularly anticlockwise
 function rotate( current ) {
-    var newCurrent = [];
-    for ( var y = 0; y < 4; ++y ) {
-        newCurrent[ y ] = [];
-        for ( var x = 0; x < 4; ++x ) {
-            newCurrent[ y ][ x ] = current[ 3 - x ][ y ];
+        var newCurrent = [];
+        for (var y = 0; y < 4; ++y) {
+            newCurrent[y] = [];
+            for (var x = 0; x < 4; ++x) {
+                newCurrent[y][x] = current[3 - x][y];
+            }
         }
```

Comment:
Not sure why you are making these changes? If you want to change the existing coding style into something else, please do so in a separate PR and do it across the whole project, not only incidentally.